cypress/integration/home.spec.ts

```diff
@@ -0,0 +1,6 @@
+describe('home page', () => {
+  it('the p tag contains the correct text', () => {
+    cy.visit('http://localhost:3000');
```

Comment:
We can also just use '/'
```suggestion
    cy.visit('/');
```

Reply:
Fixed on this [commit](https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/30/commits/4006342fb8cd9bc627cb3a1cc80b9fb474d110b5)